extensions/ql-vscode/src/view/remote-queries/CodePaths.tsx

```diff
@@ -5,9 +5,8 @@ import * as React from 'react';
 import { ChangeEvent, useRef, useState } from 'react';
```

Comment:
`CodePaths` and `FileCodeSnippet` are also meant to be re-used btw!

Reply:
I think it might be better to do that in a separate PR which also cleans up those components. Currently those files seem to be quite large, have some inconsistent parts (like inline styles) and use Primer icons. I think it would be better to clean those files up before moving them to the `common` directory so that everything in `common` and `variant-analysis` follows the proper style, uses the correct icons, etc.

Reply:
Sounds sensible!